ext/bcmath/bcmath.c

```diff
@@ -481,7 +473,7 @@ PHP_FUNCTION(bcsqrt)
 	if (bc_sqrt (&result, scale) != 0) {
 		RETVAL_STR(bc_num2str_ex(result, scale));
 	} else {
-		zend_value_error("Square root of negative number");
+		zend_argument_value_error(1, "cannot take square root of a negative number");
```

Comment:
This error message won't make sense, as it will say something like `Argument #1 cannot take square root of a negative number`.

The standard pattern here would be `"must be greater than or equal to 0"`